src/xrGame/configs_dump_verifyer.cpp

```diff
@@ -196,21 +196,16 @@ bool const configs_verifyer::verify(u8* data, u32 data_size, string256& diff)
         tmp_ini.r_string(cd_info_secion, cd_player_digest_key), tmp_ini.r_string(cd_info_secion, cd_creation_date));
 
     m_orig_config_body.w_stringZ(add_str);
+    auto hash = crypto::xr_sha1::calculate(m_orig_config_body.pointer(), m_orig_config_body.tell());
```

Comment:
I'd replaced all that `auto` with explicit `crypto::xr_sha1::hash_t`, but the `auto` is a place for possible compiler optimizations, so, probably, `auto` is better.

Reply:
@kacejot, any post-merge thoughts?)